mindsdb/integrations/handlers/dockerhub_handler/DockerHubClient.py

```diff
@@ -0,0 +1,35 @@
+import json
+import requests
+
+class DockerHubClient:
```

Comment:
Let's change the file name to be consistent to something like docker_client

Reply:
Docker Client and Docker Hub client are different.  As we are dealing with dockerhub, I would say it's better to keep it dockerhub else it will be confusing.

Reply:
Ok, let's go with `dockerhub.py`